### ISSUE TITLE
Issue with Indentation forceStrictCondition

### ISSUE THREAD

**Issue by liutikas** [checkstyle/checkstyle#613]
Using the following config
```xml
<module name="Checker">
  <module name="TreeWalker">
    <module name="Indentation">
      <property name="basicOffset" value="4"/>
      <property name="throwsIndent" value="8"/>
      <property name="lineWrappingIndentation" value="8"/>
      <property name="forceStrictCondition" value="true"/>
    </module>
  </module>
</module>
```
and the following code
```java
public class MyClass {
    public void callMe() {
        SparseArray packageIdentifiers =
                (SparseArray) mGetAssignedPackageIdentifiersMethod.invoke(
                        resources.getAssets());
    }
}
```
I get a Checkstyle error
```
MyClass.java:5: 'resources' have incorrect indentation level 24, expected level should be 16.
```

I expected not to get any errors. The error is only thrown when `forceStrictCondition` is set to `true`.

What could be causing this issue?

**Comment by romani** [comment]
It is just lack of testing, sorry, indentation is not trigerred on "invoke(".

**Comment by liutikas** [comment]
Once it is fixed, I would be glad to run a binary with a fix it through Chromium codebase to see if we hit any more snags

**Comment by romani** [comment]
please share link to sources here, we will launch over code base ourself to minimize close-reopen states.

**Comment by liutikas** [comment]
Not sure if you want to do this as Chromium code base is HUGE, but instructions are here https://code.google.com/p/chromium/wiki/AndroidBuildInstructions

Java code is mostly used for Chrome for Android.
